core-blocks/columns/index.js

```diff
@@ -23,6 +23,15 @@ import {
 import './style.scss';
 import './editor.scss';
 
+/**
+ * Allowed blocks constant is passed to InnerBlocks precisely as specified here.
+ * The contents of the array should never change.
+ * The array should contain the name of each block that is allowed.
+ * In columns block, the only block we allow is 'core/column'.
+ *
+ * @constant
+ * @type {array<string>}
```

Comment:
Unfortunate ESLint isn't smart enough here; probably because of the typing. This should be `Array` capitalized.

https://github.com/WordPress/gutenberg/blob/bf8054eb0a32f01d6a7c1caae0d3833b1f182e7d/eslint/config.js#L139-L140

Reply:
There's also the `string[]` alternative syntax.